### PR TITLE
Error handling

### PR DESCRIPTION
**Pull request by Devynnem**
____ Wrote Tests ____ Implemented ____ Reviewed

## Neccesary checkmarks
- [ ] All Tests are Passing in all environments
- [ ] The code will run locally
## Type of change
- [ ] New feature
- [x] Bug Fix
- [x] Refactor
## Description of Change:

  description:  Updated error components for bad fetch calls, wrong url, and invalid searches. Add css styling to all of those, and cleaned up various parts of the code and deleted unused code.
    
## Requested feedback: 
    
  I'd like feedback on... 

## Check the correct boxes

- [x] This broke nothing
- [ ] This broke some stuff
- [ ] This broke everything

## Testing Changes

- [ ] No Tests have been changed
- [ ] Some Tests have been changed
- [ ] All of the Tests have been changed(Please describe what in the world happened)

## Checklist:

- [ ] My code has no unused/commented out code
- [ ] My code has no binding.pry's
- [ ] I have reviewed my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have fully tested my code

(For Fun!)Please Include a link to a gif of your feelings about this branch

Link:

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -27,7 +27,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>street eatz</title>
```
Comment (Pma913) [inline public/index.html:30]: Good call on renaming! 
small detail, easily overlooked.

File: src/Components/App/App.js
Comment (Pma913) [inline src/Components/App/App.js:1]: the cleanup on this component soothes my soul

File: src/Components/App/App.js [hunk 2]
```diff
@@ -82,33 +83,25 @@ const App = () => {
...
     });
-
     return () => {
       socket.off('receive_data');
     };
   }, [])
 
   return (
-    <div>
-      <Header togView={toggleView} currentUser={currentUser}/>
```
Comment (Pma913) [inline src/Components/App/App.js:106]: I noticed this passing of unnecessary data this morning, thanks for catching that.

File: src/Components/TruckCard/TruckCard.js [hunk 3]
```diff
@@ -29,47 +28,48 @@ const TruckCard = (props) => {
...
+    } else {
+      return "open now"
+    }
   }
-}
-const currentStar = isFavorite ? Fav : notFav
-const currentUpvote = isUpvoted ? upvoted : notUpvoted
-const currentDownvote = isDownvoted ? downvoted : notDownvoted
-const starAltText = isFavorite ? "Favorited Food Truck Star" : "Not Favorited Food Truck Star"
+
```
Comment (Pma913) [inline src/Components/TruckCard/TruckCard.js:68]: clean and consistent spacing is always nice 🤙

File: src/Components/TruckCard/TruckCard.js [hunk 4]
```diff
@@ -79,17 +79,17 @@ const starAltText = isFavorite ? "Favorited Food Truck Star" : "Not Favorited Fo
       src={currentStar} 
       onClick={() => handleFav()} 
       height={50} width={50}/> 
-      <img className="image" src={props.truck.img}/>
+      <NavLink to={`/vendor/${props.truck.id}`}><img className="image" src={props.truck.img}/></NavLink>
```
Comment (Pma913) [inline src/Components/TruckCard/TruckCard.js:82]: great use of interpolation!

File: src/Components/App/App.js
Comment (emlindvall) [reply src/Components/App/App.js:1]: Same here. Looking good!

**Review by Pma913 — APPROVED** [review]
Everything looks good. Great job getting the error component up to par and thank you for taking the initiative on some much needed cleanup.

**Pull request merged**